esp/src/eclwatch/ESPLogicalFile.js

```diff
@@ -84,10 +84,15 @@ define([
             }
         },
         preProcessRow: function (item, request, query, options) {
+            var convertNull;
+            if (item.IntSize === null) {
+                convertNull = 0;
+            }
             lang.mixin(item, {
                 __hpcc_id: createID(item.NodeGroup, item.Name),
                 __hpcc_isDir: false,
-                __hpcc_displayName: item.Name
+                __hpcc_displayName: item.Name,
+                Size: convertNull //superfiles causing issues sorting by returning null
```

Comment:
You are correct this should be IntSize. I need to rework the setting to 0 logic. According to the grid there should never be a null or there will be issues sorting (as users are experiencing now). Some of the null values will appear in between the largest to smallest values.